spaces/tests.py

```diff
@@ -198,3 +195,187 @@ def test_get_space_list_view_keyerror(self):
         response = client.get("/spaces")
         self.assertEqual(response.status_code, 400)
         self.assertEqual(response.json(),{"message" : "SPACE_DOES_NOT_EXIST"})
+
+
+client = Client()
+class SpaceDetailViewTest(TestCase):
+    def setUp(self):
+        Country.objects.create(
+            name = "한국"
+        )
+        country_id = Country.objects.get(name="한국").id
+
+        User.objects.create(
+            country_id     = country_id,
+            phone          = None,
```

Comment:
```suggestion
        country = Country.objects.create(
            name = "한국"
        )


        user = User.objects.create(
            country_id     = country.id,
            phone          = None,
```
정민님 아래에 다른 코드들에도 모두 create하고 그 대상을 다시 get 하고 있으신데, 그 방법 보다는 생성한 객체를 바로 변수에 담고 그 변수를 바로 사용해주는 방법을 추천드립니다. 불 필요한 변수 선언을 줄일 수 있고, 코드도 간결해집니다 :)

Reply:
네 알겠습니다!!!